include/tesla.hpp

```diff
@@ -1059,7 +1061,33 @@ namespace tsl {
                 static PlFontData stdFontData, extFontData;
 
                 // Nintendo's default font
-                TSL_R_TRY(plGetSharedFontByType(&stdFontData, PlSharedFontType_Standard));
+                u64 languageCode;
+                if (R_SUCCEEDED(setGetSystemLanguage(&languageCode)))
+                {
+                    SetLanguage setLanguage;
+                    TSL_R_TRY(setMakeLanguage( languageCode, &setLanguage));
+                    switch (setLanguage)
+                    {
+                    case SetLanguage_ZHCN:
+                    case SetLanguage_ZHHANS:
+                        TSL_R_TRY(plGetSharedFontByType(&stdFontData, PlSharedFontType_ChineseSimplified));
+                        break;
+                    case SetLanguage_KO:
+                        TSL_R_TRY(plGetSharedFontByType(&stdFontData, PlSharedFontType_KO));
+                        break;
+                    case SetLanguage_ZHTW:
+                    case SetLanguage_ZHHANT:
+                        TSL_R_TRY(plGetSharedFontByType(&stdFontData, PlSharedFontType_ChineseTraditional));
+                        break;
```

Comment:
Have you checked if this doesn't break overlays that use the English alphabet? iirc these fonts don't include the standard glyphs